src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/hosts.e2e-spec.ts

```diff
@@ -31,9 +31,5 @@ describe('Hosts page', () => {
     it('should check at least one host is present', () => {
       hosts.check_for_host();
     });
```

Comment:
@nizamial09 changing to daemon count looks good, but can we retain the earlier inside daemon count link so that user can see each service perf_counter?

Reply:
Was that really useful for you, @sunilangadi2 ? Could you please tell us what kind of info you got/needed from there? Our impression was that the perf-counters contain little user-relevant info in a haystack of developer-meaningful metrics...

Reply:
yes agree that it's more of developer related config, also this can be seen in other part of dashboard that each daemon details page, Hence ok to remove it from here.

Reply:
Yup, those are still accessible, but we need to figure out a way to display less more meaningful info.